docs/06-using-cairo.md

```diff
@@ -0,0 +1,160 @@
+# Using Cairo and LGI
+
+These days, Awesome interface is mostly based on a library called LGI. It allows
+to access C libraries such as GTK, GLib, Cairo, Pango, PangoCairo and RSVG using
+Lua code without having to write actual "glue" C code.
+
+This is done using the GObject-introspection framework. The main advantage is
+the time saved and large number of features exposed for free. The downside is
+the lack of proper Lua contric documentation and examples. Some examples can be
+found in [LGI own documentation](https://github.com/pavouk/lgi/tree/master/docs)
+, but this doesn't even cover 1% of the actual API. Using other APIs require
```

Comment:
I think the comma is mis-misplaced. I'm not 100% sure, but doesn't this mean there will be a space before the comma?
Put differently: The comma should be at the end of the previous line.